docs/source/beta.rst

```diff
@@ -208,3 +208,134 @@ Configuring the Command Line
 With the server up and running you can now configure **qpc** to work with the server. You can do this with the ``qpc server config`` command. The ``qpc server config`` command takes a ``--host <host>`` flag and an optional ``--port <port>`` flag; defaults to ``443``. If you are using qpc on the same system where the server is running you can supply ``--host 127.0.0.1`` otherwise supply the correct IP address. If you decided to remap the port to another port you must supply that to the port option (i.e. ``--port 8443``).
 
 Now the command line has been configured you can log in with the ``qpc server login`` command. Verify your ability to log into the server.
+
+
+Getting Started
+---------------
+Now that everything is installed and configured you can begin to utilize the capabilities of Quipucords to gather information on your IT infrastructure.
+
+Quipucords inspects various sources using credentials. You must first identify what sources will be inspected from the supported list:
+
+- Network
+- vCenter server
+- Satellite
+
+Network sources are composed of IP address, IP ranges, or hostnames. vCenter server and Satellite is created with the IP address or hostname of the server. Sources additionally are composed of credentials. A Network source can have a list of credentials as it is expected that many may be needed for a broad IP range, whereas vCenter server and Satellite use a single credential.
```

Comment:
vCenter server and Satellite sources are created with either the IP address or hostname of the server.